src/components/App.tsx

```diff
@@ -35,29 +45,35 @@ const App: FC<AppProps> = ({ demo }) => {
           scope: 'launch/patient openid profile'
         })
         .then(client => {
+          // TODO: MockedFHIRClient has not mocked out requests for resources yet
+          getPatientRecord(client).then((records: fhir.DomainResource[]) => {
+            // filters out values that are empty
+            // the server might return deleted
```

Comment:
I don't have a problem with deleting these useless resources, but I don't think "deleted" is the right word for it.

Reply:
Note that this was just moved, not actually new. This was a change added a while back because we noticed some server was returning basically "empty" resources, i.e. they only had a resourceType, id, and meta. I think when we investigated the resources actually had been deleted but were still being returned in $everything for some reason.

Reply:
That makes sense and I think the description of "empty" more accurately describes it.